data/templates/firewall/nftables-nat.tmpl

```diff
@@ -73,6 +73,32 @@
 {%     set trns_addr = 'return' %}
 {%     set trns_port = '' %}
 {%   endif %}
+{#  T1083: NAT address and port translation options #}
+{%   if config.translation.options is defined and config.translation.options is not none %}
+{%     if config.translation.options.address_mapping is defined %}
```

Comment:
Is there a reason to not use:

```
{%     if config.translation.options.address_mapping is defined and config.translation.options.address_mapping == "persistent" %}
{%       set trns_opts_addr = 'persistent' %}
{%     endif %}
```

As `trns_opts_addr` is checked below anyways no need for going the extra round here - rather keep the code clean. Please also check if this would apply to the code block below.

Reply:
Thanks for tip, less code, looks cleaner